src/swap/central/exolix.ts

```diff
@@ -366,6 +374,65 @@ export function makeExolixPlugin(opts: EdgeCorePluginOptions): EdgeSwapPlugin {
       const fixedOrder = await getFixedQuote(newRequest, userSettings)
       const fixedResult = await makeSwapPluginQuote(fixedOrder)
 
+      // Limit exolix to $70k USD
+      let currencyCode: string
+      let exchangeAmount: string
+      if (newRequest.quoteFor === 'from') {
+        currencyCode = newRequest.fromCurrencyCode
+        exchangeAmount = await newRequest.fromWallet.nativeToDenomination(
+          newRequest.nativeAmount,
+          currencyCode
+        )
+      } else {
+        currencyCode = newRequest.toCurrencyCode
+        exchangeAmount = await newRequest.toWallet.nativeToDenomination(
+          newRequest.nativeAmount,
+          currencyCode
+        )
+      }
+      const data = [{ currency_pair: `${currencyCode}_iso:USD` }]
+
+      const options = {
+        method: 'POST',
+        headers: { 'Content-Type': 'application/json' },
+        body: JSON.stringify({ data })
+      }
+      let rates: RatesRespose
+      try {
+        const response = await fetchRates(fetch, 'v2/exchangeRates', options)
+        if (!response.ok) {
+          const text = await response.text()
+          throw new Error(`Error fetching rates: ${text}`)
+        }
+        const reply = await response.json()
+        rates = asRatesResponse(reply)
+      } catch (e) {
+        log.error('Error fetching rates', String(e))
+        throw new Error('Error fetching rates')
+      }
+
+      const { exchangeRate } = rates.data[0]
+      if (exchangeRate == null) throw new SwapCurrencyError(swapInfo, request)
+
+      const usdValue = mul(exchangeAmount, exchangeRate)
+      const maxExchangeAmount = div(
+        MAX_USD_VALUE,
+        exchangeRate,
+        DIVIDE_PRECISION
+      )
+      const maxNativeAmount = await newRequest.fromWallet.denominationToNative(
```

Comment:
maxNativeAmount would be incorrect for 'to' quotes. The if/else logic above identifying the exchangeAmount needs to be replicated here